config.js

```diff
@@ -58,7 +53,7 @@ const initConfig = callback => {
       config.schedule_evening_hours,
       config.schedule_evening_minutes
     );
-    callback();
+    require('./transitions').loadTransitions();
```

Comment:
Because otherwise all the tests break. I spent some time trying to work out why this would break all the tests but it's not worth me spending any more time on it now.